datascience-notebook/Dockerfile

```diff
@@ -19,16 +19,13 @@ RUN apt-get update && \
 # install Julia packages in /opt/julia instead of $HOME
 ENV JULIA_PKGDIR=/opt/julia
 
-RUN . /etc/os-release && \
-    echo "deb http://ppa.launchpad.net/staticfloat/juliareleases/ubuntu $VERSION_CODENAME main" > /etc/apt/sources.list.d/julia.list && \
-    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 3D3D3ACC && \
-    apt-get update && \
-    apt-get install -y --no-install-recommends \
-    julia && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/* && \
-    # Show Julia where conda libraries are \
-    echo "push!(Libdl.DL_LOAD_PATH, \"$CONDA_DIR/lib\")" >> /usr/etc/julia/juliarc.jl && \
+RUN mkdir -p /opt/julia-0.6.0 && \
+    wget -qO- https://julialang.s3.amazonaws.com/bin/linux/x64/0.6/julia-0.6.0-linux-x86_64.tar.gz | tar -C /opt/julia-0.6.0 -x -z --strip-components=1 -f -
```

Comment:
Thanks for the PR, @ellisvalentiner. I think we should add a check of the SHA256 of the tarball like we do for other packages coming straight from HTTP servers (e.g., https://github.com/jupyter/docker-stacks/blob/master/pyspark-notebook/Dockerfile#L19 for Apache Spark).

I can push a commit to your branch adding the sha check if you tick the "Allow edits from maintainers" in the PR.

Reply:
Good call. I'm adding the sha256 check and making a small tweak so it is easier to use different versions of Julia.